package.json

```diff
@@ -22,15 +26,17 @@
     "react-native": "*"
   },
   "dependencies": {
+    "@expo/config-plugins": "^4.0.4",
     "fbemitter": "^2.1.1"
   },
   "devDependencies": {
     "@types/fbemitter": "^2.0.32",
     "@types/react-native": "^0.57.4",
+    "expo-module-scripts": "^2.0.0",
     "tslint": "^5.11.0",
     "tslint-config-airbnb": "^5.11.0",
     "tslint-config-prettier": "^1.15.0",
-    "typescript": "^3.1.2"
+    "typescript": "4.5.5"
```

Comment:
If TS is somehow necessary for this change to work, please upgrade it separately

Reply:
PR created. I will rebase the PR after it gets merged

https://github.com/callstack/react-native-fbads/pull/314